com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/TemporalAntiAliasing.shader

```diff
@@ -221,9 +294,12 @@ Shader "Hidden/HDRP/TemporalAA"
 
 #if VELOCITY_REJECTION
             float lengthMV = motionVectorLength * 10;
```

Comment:
Whats the meaning of that *10 for the motion vector length?

Reply:
A mix of empirically set value and better space to store it in the limited bit range of the output history. 

I shall probably add a comment